appserver/extras/docker-images/server-node/src/main/docker/bin/entrypoint.sh

```diff
@@ -151,11 +152,13 @@ function createNewInstance {
         fi
     fi
 
-    # Register Docker container ID to DAS
-    echo "Setting Docker Container ID for instance ${PAYARA_INSTANCE_NAME}: ${DOCKER_CONTAINER_ID}"
-    ASADMIN_COMMAND="${ASADMIN} -I false -H ${PAYARA_DAS_HOST} -p ${PAYARA_DAS_PORT} -W ${PAYARA_PASSWORD_FILE} _set-docker-container-id --instance ${PAYARA_INSTANCE_NAME} --id ${DOCKER_CONTAINER_ID}"
-    echo "${ASADMIN_COMMAND}"
-    ${ASADMIN_COMMAND}
+    if [ -z "${DOCKER_CONTAINER_ID}" ]; then
```

Comment:
Isn't this backwards? You're setting the container ID if it isn't present, rather than if it is.

Reply:
oops, how did that copy and paste went wrong? Thx @Pandrex247